app/assets/javascripts/gradebook.js

```diff
@@ -11,6 +11,7 @@ var slickgrid_options = {
   defaultColumnWidth: 65,
   syncColumnCellResize: true,
   fullWidthRows: true,
+  enableTextSelectionOnCells: true,
```

Comment:
As noted, text selection disappears immediately after mouse up.

One fix that I tested as working is: https://github.com/mleibman/SlickGrid/pull/936

Reply:
Thanks for bringing up a fix. I think at some point maybe moving away from slickgrid would be good because it's not really in development anymore

Reply:
That would be good, although most libraries I see are not really in development either